uptrain/tests/test_measurables_accuracy.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pytest
+
+from uptrain.core.classes.measurables import (
```

Comment:
Hey @a-r-r-o-w , do we need this file now? ADWIN already tests for MAE. I think we can also add MAPE there? And DDM tests for AccuracyMeasurable

Reply:
Yes, we do not require this file since it does not test the entire pipeline in itself, which is our goal when writing these tests. I've removed it and made other discussed changes.

The only thing that remains is fixing the plot bug which can be seen on the dashboard when running the current tests in this PR.